.coveragerc

```diff
@@ -90,6 +90,9 @@ omit =
     homeassistant/components/eight_sleep.py
     homeassistant/components/*/eight_sleep.py
 
+    homeassistant/components/ecoal_boiler/*
```

Comment:
Update the path so that it matches the component module.

Reply:
Done.